portfolio/src/main/webapp/script.js

```diff
@@ -80,13 +80,68 @@ function openCity(evt, sectionName) {
 /**
  * Fetch function from servlet
  */
-const START_INDEX = 4;
-const END_INDEX = 17;
 async function getWelcomeData() {
-  const response = await fetch('/data');
-  let quote = await response.text();
-  // Slice quote so that formatting remains when on navigates to url/data
-  document.getElementById('data-container').innerText = quote.slice(START_INDEX, END_INDEX);
+    // Get comments from the server
+    const response = await fetch('/data');
+    let comments = await response.json();
+
+    // Display the comments
+    const commentsEl = document.getElementById('comment_list');
+    comments.forEach((comment) => {
+        commentsEl.appendChild(createCommentElement(comment));
+    });
+}
+
+/**
+ * Creates one comment element
+ */
+function createCommentElement(comment) {
+    const commentElement = document.createElement('li');
+    commentElement.className = 'comment';
```

Comment:
`className` is deprecated, I think.  You should probably use `commentElement.classList.add('comment')` instead.